### PR TITLE
Feat: added card component

### PR DESCRIPTION
**Pull request by Manpreet-hub**
Added card component
- Cards/Horizontal Card
- Card with badge
-  Card with Text Overlay
chrome_r4tpVu9hjk

### CODE REVIEW

**Review by SukanyaSen530 — APPROVED** [review]
👍🏼 GTG

**Comment by Radheshyam1234** [comment]
I would suggest that instead of adding font-weight to individual element by writing css . Just define  classes like font-weight bold, 
font-weight semibold, font-weight regular , so that you can use it anywhere.

File: Styles/utility.css [hunk 1]
```diff
@@ -62,6 +62,7 @@ li{
     align-items: center;
     width: 100%;
     flex-wrap: wrap;
+    gap: 4%;
```
Comment (tanishk15sharma) [inline Styles/utility.css:65]: suggestion , you can use `rem` here

**Review by tanishk15sharma — APPROVED** [review]
good to merge

File: Styles/utility.css (hunk 1 above)
Comment (Manpreet-hub) [reply Styles/utility.css:65]: Thanks!  will consider this utility.css changes in next  PR.

**Pull request merged**